source/infra/service-continuity.html.md

```diff
@@ -53,7 +53,7 @@ The following table details the high-level plan for each service. A link to a de
 | DLRN instance for upstream rpm-packaging | Medium | Built repos can be synced using rsync to https://trunk.rdoproject.org (in the offsite cloud). In case of an outage in the RDO cloud, we can switch the DNS entry to the machine in trunk.rdoproject.org (maybe some vhost is needed here????). | |
 | images.rdoproject.org (+ CI docker registry) | High | Rsync to machine in offsite cloud. In case of an outage in the RDO cloud, we can switch the DNS entry to the machine in the offsite cloud. ||
 | www.rdoproject.org | High | Web contents are static and stored in <https://github.com/redhat-openstack/website> . In case of an outage, we can switch the DNS entry to a machine in the offsite cloud. The builder is hosted by OSAS/OSCI in the Community Cage and the RDO infra team has root access. Both can be fully recreated using Ansible | For rdo-release.rpm rdoproject.org redirects to <https://repos.fedorapeople.org/openstack/>. This is out of RDO infra but is used by different CI jobs and users. *I assume repos.fedorapeople.org has its own replication/DR, right?*|
-| blogs.rdoproject.org | Medium | Previous Enovance blog, still has content published from time to time. Was migrated from the old/unmaintained machine and WordPress to RDO Cloud with a recent WP version. It was installed manually, thus automation needs to be done ||
+| blogs.rdoproject.org | Medium | Previous Enovance blog, still has content published from time to time. Was migrated from the old/unmaintained machine and WordPress to RDO Cloud and again on 2019-09-17 to WP hosting. ||
```

Comment:
I indeed forgot to update this document. Could you add in parenthesis that it's managed by OSPO/OSCI please?